script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_5.txt

```diff
@@ -38,7 +38,7 @@ It hasn't changed at all since then.
 According to my father, the house was heavily renovated after the war, but the basic structure was inherited from my grandfather's and his grandfather's generation.
 <Page13>
 This is the castle of a powerful family who has been in this land for more than hundreds of years.
-For Makihisa... no, even to the people related to the Tohno, this mansion is a sanctuary.
```

Comment:
Is this definitely the 'have a connection' meaning over the attend / participate (i.e, "For Makihisa... no, for all members of the Tohno family..." meaning? 
> 槙久……いや、遠野に連なる人々にとって、この屋敷は聖域そのものだ。